src/analysis/special_functions/trigonometric.lean

```diff
@@ -2864,20 +2864,20 @@ lemma differentiable.clog {f : E → ℂ} (h₁ : differentiable ℂ f)
 
 end log_deriv
 
-section chebyshev₁
+namespace chebyshev
 
 open polynomial complex
 
 /-- The `n`-th Chebyshev polynomial of the first kind evaluates on `cos θ` to the
 value `cos (n * θ)`. -/
-lemma chebyshev₁_complex_cos (θ : ℂ) :
-  ∀ n, (chebyshev₁ ℂ n).eval (cos θ) = cos (n * θ)
-| 0       := by simp only [chebyshev₁_zero, eval_one, nat.cast_zero, zero_mul, cos_zero]
-| 1       := by simp only [eval_X, one_mul, chebyshev₁_one, nat.cast_one]
+lemma T_complex_cos (θ : ℂ) :
```

Comment:
Do you want this to be called `polynomial.chebyshev.T_complex_cos` or `chebyshev.T_complex_cos`? Your other file seems to define `chebyshev.T`, suggesting the latter; but maybe it is the other file that is in error.

Reply:
I will go for `polynomial.chebyshev.T_complex_cos` and similarly for the other theorems. I imagine if one wants to use these, one would want to `open polynomial` anyway as one would want to be able to just write `X` for the indeterminant.